core/src/main/scala/besom/internal/logging.scala

```diff
@@ -353,6 +354,28 @@ object logging:
       .replace()
   }
 
+  /** For tests only
+    *
+    * @param handler
```

Comment:
I can't see this param, there's level and maxLines

Reply:
borked, true